llvm/lib/Target/RISCV/RISCVTargetTransformInfo.cpp

```diff
@@ -2329,6 +2329,15 @@ unsigned RISCVTTIImpl::getMaximumVF(unsigned ElemWidth, unsigned Opcode) const {
   return std::max<unsigned>(1U, RegWidth.getFixedValue() / ElemWidth);
 }
 
+TTI::AddressingModeKind
+RISCVTTIImpl::getPreferredAddressingMode(const Loop *L,
+                                         ScalarEvolution *SE) const {
+  if (ST->hasVendorXCVmem())
+    return TTI::AMK_PostIndexed;
+
+  return TTI::AMK_None;
```

Comment:
Instead of returning `TTI::AMK_None`, I should probably call the generic `getPreferredAddressingMode` implementation?

Reply:
Please do that.